### PR TITLE
Updated the blog post.

### PR DESCRIPTION
**Pull request by wermos**
Added some things to the blog post and wrote about the post-midterm work.

### CODE REVIEW

**Comment by wermos** [comment]
Ping @niermann999 for review.

File: _gsocblogs/2022/blog_Acts_TirthankarMazumder.md [hunk 1]
```diff
@@ -117,14 +117,60 @@ Apart from my work on Fast5×5, I also fired off a [small PR](https://github.com
...
+<center><caption>Benchmark plot for the bound coordinates to free coordinates transform with <code>double</code> matrices.</caption></center>
+
+It is clear from these plots that Fastor is consistently near the top (i.e. either the fastest or the second-fastest in all of these tasks).
+
+## Integrating Fastor into `algebra-plugins`
+
+I first had to define the math functions that live in the [`math` subdirectory](https://github.com/wermos/algebra-plugins/tree/528327c51d46cf198fe79fd83856c47b29d6d2b1/math/fastor) in terms of `Fastor` primitives and functions. After that, I had to declare the standard litany of types in the [`storage` subdirectory](https://github.com/wermos/algebra-plugins/tree/528327c51d46cf198fe79fd83856c47b29d6d2b1/storage/fastor). Then, I had to define the [`fastor_fastor` frontend](https://github.com/wermos/algebra-plugins/tree/528327c51d46cf198fe79fd83856c47b29d6d2b1/frontend/fastor_fastor) so that the backend is actually usable and the tests can be written.
```
Comment (niermann999) [inline _gsocblogs/2022/blog_Acts_TirthankarMazumder.md:166]: This seems a bit incoherent to me, tbh. I had hoped for a final wrap-up of the project, along the lines of a bit of context of the project, a short reflection on the benchmarks (Eigen vs blaze vs fastor etc). Maybe a short introduction to fastor? OK, there is a more detailed overview about algebra-plugins in your blog...

Comment (niermann999) [reply _gsocblogs/2022/blog_Acts_TirthankarMazumder.md:166]: Just telling the overarching story of the project

**Review by niermann999 — APPROVED** [review]

**Comment by niermann999** [comment]
Ah, Ok, sorry, found it in your GSoC submission!

**Review by agheata — APPROVED** [review]
LGTM

**Pull request merged**